packages/api/src/services/actions.ts

```diff
@@ -79,6 +79,17 @@ export function Handlers(config: Config, appState: Dependencies): Actions {
       // convert this to tuple to save bytes.
       return results.map(({ price, timestamp }) => [timestamp, price]);
     },
+    async getEmpStats(address: string, currency: CurrencySymbol = "usd") {
```

Comment:
what is the convention with documenting these kinds of functions with a natspec type thing? in other places in the protocol repo we document the function's intention, parameters, and return type. that might be useful here seeing this is an exported function that will be called from a lot of places. I think there are comment formatting types that will expose the comments when importing modules making working with packages much easier as your IDE will show you these definition comments import. thoughts?

Reply:
why does this function need to be async? is the `latest.get(address)` async?

Reply:
> why does this function need to be async? is the latest.get(address) async?

I guess it technically doesn't have to be, its just for consistency and i dont think it hurts anything. `latest.get(address)` is async but the parent function doesnt need to be.

Reply:
> what is the convention with documenting these kinds of functions with a natspec type thing? in other places in the protocol repo we document the function's intention, parameters, and return type.

yea this is tricky, these are meant to be functions exposed through some transport, in this case express. A user of the api would not necessarily have access to comments in this form. So from that perspective the comments would not be that useful and would be costly to maintain. This is what's great about graphql, because it gives a really useful discovery mechanism for api calls out of the box. 

What ive done in this case is documented all calls in the `api/README.md` and created example calls through [postman](https://www.getpostman.com/collections/e99f4e09e2443cb31ef4). though its not as thorough or formal as a natspec, but i feel like the readme / example calls are good enough for someone using the api. 

Would really love to be able to somehow compile the typescript into docs which could be put on a page or served through the api, like graphql.  Happy to get more discussion around this though!